packaging/make-omnibus

```diff
@@ -89,31 +89,22 @@ else
     mkdir $INSTALLDIR
 fi
 
-NODE_VERSION="v0.10.45"
-NODE_SOURCE="http://nodejs.org/dist/${NODE_VERSION}/node-${NODE_VERSION}.tar.gz"
-
-node_src() {
-    echo "Checking for node-$NODE_VERSION source"
+[[ "$PLATFORM" == "i386" ]] && NODE_ARCH="x86"
+[[ "$PLATFORM" == "x86_64" || "$PLATFORM" == "amd64" ]] && NODE_ARCH="x64"
+NODE_VER="4.4.5"
+NODE_TGZ="node-v${NODE_VER}-linux-${NODE_ARCH}.tar.gz"
```

Comment:
We technically support non-Linux platforms in this script.  Down in make_package() we have cases for EL and Ubuntu but failing that we make a tarball, so I'd say we should either drop the facade of omnibus being possible on non-Linux or make it a bit smarter about platform selection for the binary download.